spk/java-11-openjdk/Makefile

```diff
@@ -59,4 +66,4 @@ include ../../mk/spksrc.spk.mk
 .PHONY: $(SPK_NAME)_extra_install
 $(SPK_NAME)_extra_install:
 	cd $(STAGING_DIR)/bin ; \
-	ln -sf ../jvm/openjdk-11.0.12/bin/* .
+	ln -sf ../jvm/openjdk-$(SPK_VERS).$(JAVA_MINOR)/bin/* .
```

Comment:
I propose to use the full version as SPK_VERS
SPK_VERS = 11.0.12

Reply:
Are you sure DSM7 supports 3 digits?
I think i came by a regex check where it must be 2 digits.
(i think DSM7 writes the regex in /var/log/packages/<my package>.log if it is invalid)
my first version i compiled i named 11.0.12+7 (ofc the + was invalid but the regex it printed looked like  digit dot digit).

Reply:
Yes I am sure, the regex is `[^\d+(\.\d+){0,5}(-\d+)?$]` so it supports up to 6 numbers separated by `.`.
We already have erlang 23.3.4.1-2  (4 numbers) that runs on DSM7.

Reply:
Great. Then I prefer to have the SPK_NAME / SPK_VERS as

NVR = java-11-openjdk-11.0.12.7-1 if you do not mind ?
Java version 11.0.12 build 07 release 1.